serviceAccessConfig/etc/init.d/serviceAccessConfig

```diff
@@ -0,0 +1,84 @@
+#! /bin/sh
+# Copyright 2016 SUSE LLC All Rights Reserved.
+#
+#  This file is part of serviceAccessConfig
+#
+#  This program is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 3 of the License, or
+#  (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+### BEGIN INIT INFO
+# Provides:          cspInfraServerAccessConfig
+# Required-Start:    $apache2
```

Comment:
To the best of my knowledge insserv or chkconfig do not error out is the service is not on the system, i.e. if it is an haproxy system and apache is not installed this is OK. I did add haproxy to required start as support for both is implemented.

Also whild looking at this I noticed the cut and paste error in Provides: which I fixed as well.